org.eclipse.xtext/src/org/eclipse/xtext/resource/OutdatedStateManager.java

```diff
@@ -68,7 +69,7 @@ public void checkCanceled(ResourceSet rs) {
 			}
 			return work.exec(param);
 		} catch (Throwable e) {
-			throw new RuntimeException(e);
+			throw Exceptions.sneakyThrow(e);
```

Comment:
I'd prefer org.eclipse.xtext.util.Exceptions.throwUncheckedException(Throwable) which does not throw a checked exception without declaring it.

Reply:
@szarnekow can you please create a PR. i dont get this done.

Reply:
@szarnekow updated the PR